### PR TITLE
 Houdini:add product types table

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description

Adding a list of stock product types that come with Ynput's Houdini addon.

Here's a preview:
image

### CODE REVIEW

**Review by moonyuet — APPROVED** [review]

**Comment by MustafaJafar** [comment]
Hey @BigRoy , 
Do you have any opinion that you would like to share with us regarding this PR?

File: website/docs/addon_houdini_admin.md [hunk 1]
```diff
@@ -126,6 +126,62 @@ Disable this if you want the publisher to ignore start and end handles specified
...
+| `Redshift ROP` | `redshift_rop` | Redshift Render. |
+| `Review` | `review` | Review with OpenGL ROP. |
+| `Static Mesh (FBX)` | `staticMesh` | Static Meshes as FBX. |
+| `USD (experimental)` | `usd` | Universal Scene Description. |
+| `USD Render (experimental)` | `usdrender` | USD Render. |
+| `VDB Cache` | `vbdcache` | OpenVDB from Geometry ROP. |
+| `VRay ROP` | `vray_rop` | VRay Render. |
+
+* **Product Type Label:** It's mostly used in AYON UI e.g. Publisher Tool.
+* **Product Type Name:** The name used in the background. AYON Settings expects you to use this name not the label.
```
Comment (BigRoy) [inline website/docs/addon_houdini_admin.md:183]: Is there a reason the table is duplicated?

Comment (MustafaJafar) [reply website/docs/addon_houdini_admin.md]: Oh, That's happened by accidents few minutes ago. 

Should be fixed here https://github.com/ynput/ayon-documentation/pull/163/commits/f1ff6c0850b51788750fb3ed6bbb8364fca48846

Thanks!

**Review by BigRoy — COMMENTED** [review]
I suppose having a list is helpful - whether we're able to keep it sync with the addon nicely and in a timely manner is hard to say. 

Only thing that stood out to me is usage of `PointCache` instead of `Pointcache` - not sure if we need the uppercase `C` there.

Probably best left to someone else to approve it that can look at this from a higher POV. :)

**Comment by MustafaJafar** [comment]
> Only thing that stood out to me is usage of `PointCache` instead of `Pointcache` - not sure if we need the uppercase `C` there.

I copied pasted the `label` and `product_type` 

https://github.com/ynput/ayon-core/blob/df976a4ab8900b14c70331519afd563e871c8d09/client/ayon_core/hosts/houdini/plugins/create/create_bgeo.py#L12-L13

> whether we're able to keep it sync with the addon nicely and in a timely manner is hard to say.

That's a good question!

**Comment by MustafaJafar** [comment]
mentioning @antirotor .. This PR is meant to align with https://github.com/ynput/ayon-core/pull/9 to help admins know which values to use with which product types.

**Comment by mkolar** [comment]
> mentioning @antirotor .. This PR is meant to align with [ynput/ayon-core#9](https://github.com/ynput/ayon-core/pull/9) to help admins know which values to use with which product types.

I made a comment on the corresponding PR as well. I'm not sure this is that useful. We have a page with all the product types in the global docs section and then attributes on the addon in the docs that is visible here https://ayon.ynput.io/features?addons=houdini

update that, but don't add the same information into a new table

**Comment by MustafaJafar** [comment]
This is no longer needed.

**Pull request closed without merging**